src/storybook/stories/Section.stories.tsx

```diff
@@ -58,7 +54,7 @@ const SectionMeta: Meta<SectionMetaProps> = {
 export default SectionMeta;
 
 export const ListedSectionItems: Meta<SectionMetaProps> = {
-  args: {withFullPadding: true, backgroundColor: 'background_2'},
+  args: {style: {margin: 12}, backgroundColor: 'background_2'},
```

Comment:
Not sure if anyone has looked into how we can choose a size from `theme.spacings` in storybook?

Reply:
This is what I'm wondering as well, I looked into the other storybook but they use these kind of styling so I haven't looked into it.